### PR TITLE
SM-97 Hand Gesture Paint Page

### PR DESCRIPTION
**Pull request by toha87**

### CODE REVIEW

File: src/containers/GestureDemo/GesturePaintDemo.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+const GesturePaintDemo = () => {
+
+    const HandGesturesHook = useHandGestures();
+
+    return (
+        <div style={{background: "black", height: "100vh"}}>
+            {
+                HandGesturesHook.paintUI
+            }
+        </div>
```
Comment (blazinaj) [inline src/containers/GestureDemo/GesturePaintDemo.js:21]: ❤️ the hook

File: src/hooks/useHandGestures.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    const canvas = document.getElementById('video-canvas');
+    const context = canvas && canvas.getContext("2d");
+
+    const [ctx, setCtx] = useState(null);
+
+    const modelParams = {
+        flipHorizontal: true, // flip e.g for video
+        imageScaleFactor: 0.7, // reduce input image size for gains in speed.
+        maxNumBoxes: 20, // maximum number of boxes to detect
+        iouThreshold: 0.5, // ioU threshold for non-max suppression
```
Comment (blazinaj) [inline src/hooks/useHandGestures.js:21]: What's ioU stand for?

File: src/hooks/useHandGestures.js [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+
+            ctx.lineWidth = 10;
+            ctx.lineCap = 'round';
+            ctx.strokeStyle = 'white';
+
+            ctx.moveTo(x, y); // from
+
+            ctx.lineTo(x, y); // to
+
+            ctx.stroke(); // draw it!
```
Comment (blazinaj) [inline src/hooks/useHandGestures.js:107]: This is awesome!

**Review by blazinaj — APPROVED** [review]
This looks really promising and clean. I'm surprised how little code it took to do this.

File: src/hooks/useHandGestures.js (hunk 2 above)
Comment (toha87) [reply src/hooks/useHandGestures.js:21]: IOU calculation is actually used to measure the overlap between two proposals. Wiki :)

**Pull request merged**